repositories.yaml

```
@@ -1568,3 +1568,25 @@ applications:
     channels:
       - v1_name: tiktokreporter-android
         app_id: org.mozilla.tiktokreporter
+
+  - app_name: firefox_bridge
+    canonical_app_name: Firefox Bridge
+    app_description: |
+      This is a chromium & firefox extension that allows the user to easily swap
+      between browsers on their device.
```

we should slightly edit this, it's what shows up in the dictionary:

```suggestion
      A Chromium & Firefox extension that allows the user to easily swap
      between browsers on their device.
```